pandas/io/html.py

```diff
@@ -116,20 +116,31 @@ def _get_skiprows(skiprows):
                     type(skiprows).__name__)
 
 
-def _read(obj):
+def _read(obj, http_params=None):
     """Try to read from a url, file or string.
 
     Parameters
     ----------
     obj : str, unicode, or file-like
 
+    http_params : dict or requests.Session(), default None
```

Comment:
so again put this template in pandas.io.common, then you can import it and add it rather than copy-pasting all over the place.